### ISSUE TITLE
Support for Multimedia in Posts

### ISSUE THREAD

**Issue by gonzofish** [gonzofish/talc#4]
As the title says, there should be support for including images, videos, etc. for posts

The images/videos need a directory where they live (which should be configurable) and that copies over any files mentioned in any template during `build`.

**Comment by gonzofish** [comment]
Maybe something like this would work:

```html
<link href="<!-- talc:static:file/path.css -->" type="stylesheet">
<img src="<!-- talc:static:deeper/file/path.jpg -->">
```

`talc:static` would be a special variable that knows to look at `config.static` and prepend it to the front of the value after the last `:`.

So if `config.static = 'my-static/dir'`, the above would compile to:

```html
<link href="my-static/dir/file/path.css" type="stylesheet">
<img src="my-static/dir/deeper/file/path.jpg">
```

**Comment by gonzofish** [comment]
Closed by #33
